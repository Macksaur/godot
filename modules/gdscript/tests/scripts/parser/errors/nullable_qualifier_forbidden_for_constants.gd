func test():
    const CONSTANT: String? = null # Should error
