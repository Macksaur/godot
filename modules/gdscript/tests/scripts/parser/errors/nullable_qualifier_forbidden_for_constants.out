GDTEST_PARSER_ERROR
Nullable qualifier ("?") not allowed in this context
